GUI/DeusEx/Classes/MenuScreenRandoOptChoices.uc

```diff
@@ -0,0 +1,29 @@
+class MenuScreenRandoOptChoices expands MenuUIMenuWindow;
+
+defaultproperties
+{
+     ButtonNames(0)="Randomizer"
+     ButtonNames(1)="Visuals"
+     ButtonNames(2)="Audio"
+     ButtonNames(3)="Gameplay"
+     ButtonNames(4)="?????"
```

Comment:
could adjust advanced options of the current game, grayed out when not in a game

would require some changes to how that menu acquires the DXRFlags